devtools/test_dashboard/style.css

```diff
@@ -61,4 +61,5 @@ header span{
 }
 .dashboard-plot{
   margin-bottom: 30px;
+  width: 1100px;
```

Comment:
There's code in `devtools.js` that actually resizes the `div.dashboard-plot` when you search to fill the full width - I wasn't sure how best to deal with it and I figured most mocks would have dimensions set, but we may want to change that.